canonical/ubuntu.com
# /download/desktop 24.04 copy updates

## Done

- Applied copy updates as per [doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13722.demos.haus/download/desktop
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the needed changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10051
